Intersect.Client/Core/Fade.cs

```diff
@@ -73,12 +90,22 @@ public static void Update()
                 {
                     sCurrentAction = FadeType.None;
                     sFadeAmt = 255f;
+
+                    if (InformServer)
+                    {
+                        InformServerOfCompletion();
+                    }
                 }
             }
 
             sLastUpdate = Timing.Global.MillisecondsUtc;
         }
 
+        private static void InformServerOfCompletion()
+        {
+            PacketSender.SendFadeCompletePacket();
+            InformServer = false;
```

Comment:
set `InformServer = false` before sending the packet after the check, we don't want it lingering because if this segment of the call stack crashes and we add recovery code above it in the call stack, we could end up just repeatedly spamming the server